mcap_vendor/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ macro(build_mcap_vendor)
   include(FetchContent)
   fetchcontent_declare(mcap
     GIT_REPOSITORY https://github.com/foxglove/mcap.git
-    GIT_TAG a06ec974f21b47f9f8425f7e70a599e3aa116d95 # releases/cpp/v0.1.2
+    GIT_TAG releases/cpp/v0.2.0
```

Comment:
I believe it’s common / part of some ROS style guide to use a SHA here to ensure consistency even if the tag were to change (which technically can happen)

Reply:
Thanks, I was wondering why that was there before.

Reply:
> I believe it’s common / part of some ROS style guide to use a SHA here to ensure consistency even if the tag were to change (which technically can happen)

Correct, we've switched to using the SHA everywhere in the ROS 2 core vendor libraries.

Reply:
I personally don't mind using a tag - the SHA is probably more bulletproof though. Ideally even if the tag is overwritten by some cowboy tagger, it would not be with a changed API. Using the tag would allow for "free" patch inclusions if a less-specified version tag was used, too, such as `v0.2` or `v0.2.x`

Reply:
those kinds of tags might be controversial - git itself is highly opinionated about tags but i've seen a lot of github projects, especially Actions, use rolling tags a la Docker to allow for free patch updates. I suppose a branch would be the more git-approved way of going about this, rather than a tag, like GitFlow style?